src/recorder.h

```diff
@@ -75,6 +79,12 @@ class Recorder {
   /// returns the unique id associated with this cyclus simulation.
   boost::uuids::uuid sim_id();
 
+  /// returns whether or not the unique simulation id will be injected.
+  bool inject_sim_id() { return inject_sim_id_; };
+
+  /// sets whether or not the unique simulation id will be injected.
+  void inject_sim_id(bool x) { inject_sim_id_ = x; };
```

Comment:
This needs to call `set_dump_count` or the datums will be generated wrong - and the resizing could result in random junk being recorded to database.